dev/devicelab/manifest.yaml

```diff
@@ -811,3 +811,18 @@ tasks:
   #   stage: devicelab_ios
   #   required_agent_capabilities: ["mac/ios", "ios/gl-render-image"]
   #   flaky: true
+
+  new_gallery_chrome_run_test:
```

Comment:
Would it be better to call these gallery_v2_chrome_run_test ?

Reply:
Done: changed to `flutter_gallery_v2_chrome_run_test`.